src/main/java/uk/gov/justice/digital/service/DomainService.java

```diff
@@ -100,8 +99,14 @@ private void refreshCDCRecord(SparkSession spark, DomainDefinition domainDefinit
 
                             logger.info("Saving transformed CDC record of table " + tableName);
                             executor.saveDomain(spark, transformedDataFrame, domainName, tableDefinition, operation);
-                        } catch (DomainExecutorException | DataStorageException e) {
-                            logger.warn("Failed to process domain for record", e);
+                        } catch (Exception e) {
```

Comment:
This ensures the CDC job does not fail because any internal exception such as a NullPointerException.